executor/runtime/types/pod.go

```diff
@@ -746,6 +755,34 @@ func (c *PodContainer) extractServiceMesh() error {
 	return nil
 }
 
+func (c *PodContainer) parsePodCommandAndArgs() error {
```

Comment:
Can you add a docstring for why we do this?
I think your commit message has most of it

Reply:
Good idea - done.